src/bin/CommandPolykey.ts

```diff
@@ -65,6 +65,12 @@ class CommandPolykey extends commander.Command {
       this.exitHandlers.errFormat = opts.format === 'json' ? 'json' : 'error';
       // Set the logger according to the verbosity
       this.logger.setLevel(binUtils.verboseToLogLevel(opts.verbose));
+      // Set the logger according to the format
```

Comment:
Should be `Set the logger formatter according to the format`.

Reply:
This should be done on `src/bin/polykey-agent.ts` too.

Reply:
These changes have been made now